src/site/_includes/components/Author.js

```diff
@@ -24,22 +24,23 @@ module.exports = ({id, showSocialMedia = false, small = false}) => {
     throw new Error(
       `Can't create Author component for "${id}" without contributor ` +
         `information. Please check '_data/contributors.js' and make sure the ` +
-        `author you provide is a key in this object.`,
+        'author you provide is a key in this object.',
```

Comment:
is it possible to use backticks here as well? seems like it might be good to be consistent with the two lines above in case someone goes in to change this block in the future